controllers/cloudflareCache.js

```diff
@@ -0,0 +1,92 @@
+const logsQuery = require("../models/logs");
+const cloudflare = require("../services/cloudflareService");
+const { logType } = require("../constants/logs");
+const { MAX_CACHE_PURGE_COUNT } = require("../constants/cloudflareCache");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+/**
+ * Purges the Cache of Members Profile Page
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const purgeCacheByUserOrSuperUser = async (req, res) => {
```

Comment:
can we rename it to just purgeCache, it it were a separate controller it could have been purgrCacheUser and purgeCacheSuperuser

Reply:
There's already a function present named as purgeCache